core/src/main/scala/org/apache/spark/ui/jobs/StageTable.scala

```diff
@@ -200,10 +200,10 @@ private[ui] class StagePagedTable(
       stageHeadersAndCssClasses.map { case (header, tooltip, sortable) =>
         val headerSpan = tooltip.map { case (title, left) =>
           if (left) {
-            /* Place the shuffle write tooltip on the left (rather than the default position
+            /* Place the shuffle write tooltip on the auto (rather than the default position
```

Comment:
Nit: we need to reword this comment: "Place the tooltip automatically rather than on top because..."
Actually you don't need the if-else here anymore?
It might be worth double-checking that the auto placement on this element works well.

Reply:
@07ARB I think we can make one more change here to simplify.

Reply:
@srowen, auto is not working properly ,tooltips will display always in top-left corner of browser (chrome).
![Screenshot 2019-11-06 at 8 22 32 AM](https://user-images.githubusercontent.com/8948111/68264808-27113e00-0070-11ea-95fa-e46a1a26ab91.png)

Reply:
That's weird! OK then let's standardize on anything that displays reasonably everywhere. I think a few of these worked differently on purpose to avoid display issues though; maybe we leave any special logic as-is.